ZipBuilder/Sources/FirebaseReleaser/main.swift

```diff
@@ -41,6 +41,11 @@ struct FirebaseReleaser: ParsableCommand {
           help: "Initialize the release branch")
   var pushOnly: Bool
 
+  /// Set this option to update tags only.
+  @Option(default: false,
+          help: "Update the tags")
+  var updateTags: Bool
```

Comment:
Maybe `updateTagsOnly` to be consistent with `pushOnly`  and corresponding help string?

Reply:
Done.